cime_config/config_compsets.xml

```diff
@@ -43,6 +43,16 @@
     <lname>1850_CAM60_CLM50%BGC-CROP_CICE_POP2%ECO_MOSART_CISM2%GRIS-NOEVOLVE_WW3_BGC%BDRD</lname>
   </compset>
 
+  <compset>
+    <alias>BLT1850_v0b</alias>
```

Comment:
We are changing the name of our F HIST compsets to be "v0c".  This change is necessary because the longname is changing for FMTHIST with our addition of %GHGMAM4.  If you want the CESM B compsets to match the name of the F compsets, you will need to change them to be "v0c"